block-storage/src/main/scala/coop/rchain/blockstorage/deploy/DeployStorage.scala

```diff
@@ -0,0 +1,14 @@
+package coop.rchain.blockstorage.deploy
+
+import coop.rchain.casper.protocol.DeployData
+import coop.rchain.crypto.signatures.Signed
+
+trait DeployStorage[F[_]] {
+  def put(deploys: List[Signed[DeployData]]): F[Unit]
```

Comment:
Maybe _add_ or _append_ would be more appropriate name in this case? My intuition for _put_ is to replace state.